tests/Fixtures/TestBundle/Document/MultiRelationsDummy.php

```diff
@@ -89,7 +89,7 @@ public function addOneToManyRelation(MultiRelationsRelatedDummy $relatedMultiUse
 
     public function getNestedCollection(): Collection
     {
-        return $this->nestedCollection;
+        return $this->nestedCollection->map(fn ($entry) => ['name' => $entry->name]);
```

Comment:
I'm not fond of this why is this necessary?

Reply:
this mapping function was neccessary because mongo ODM behaves different like ORM does. The ORM Entity provides a Collection with each entry being an associative array. The ODM Document on the other hand provides a Collection with each entry being an instance of `MultiRelationsNested` / `MultiRelationsNestedPaginated`. So my intention here was to align the Testables here to get the same results.

Without this mapper the case for `MultiRelationsNestedPaginated` gets broken in test. If this is not acceptable for you, you may undo this mapping and find another way - I was not able to.